apps/api-gateway/src/platform/platform.controller.ts

```diff
@@ -1,18 +1,18 @@
 import { Controller, Get, HttpStatus, Logger, Param, Query, Res, UseFilters, UseGuards } from '@nestjs/common';
 import { PlatformService } from './platform.service';
-import { ApiBearerAuth, ApiExcludeEndpoint, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { ApiBearerAuth, ApiExcludeEndpoint, ApiOperation, ApiQuery, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { ApiResponseDto } from '../dtos/apiResponse.dto';
 import { GetAllSchemaByPlatformDto } from '../schema/dtos/get-all-schema.dto';
 import { IUserRequestInterface } from '../interfaces/IUserRequestInterface';
 import { User } from '../authz/decorators/user.decorator';
 import { Response } from 'express';
 import { ISchemaSearchPayload } from '../interfaces/ISchemaSearch.interface';
-import IResponseType, { IResponse } from '@credebl/common/interfaces/response.interface';
+import { IResponse } from '@credebl/common/interfaces/response.interface';
 import { ResponseMessages } from '@credebl/common/response-messages';
 import { CustomExceptionFilter } from 'apps/api-gateway/common/exception-handler';
 import { AuthGuard } from '@nestjs/passport';
 import * as QRCode from 'qrcode';
-
+import { SortFields } from '../schema/enum/schema.enum';
```

Comment:
any specific reason to use schema enum in platform service?

Reply:
Resolved